src/gui/processing/qgsprocessingmodelerwidget.h

```diff
@@ -0,0 +1,164 @@
+/***************************************************************************
+                         qgsprocessingmodelerwidgetwrapper.h
```

Comment:
qgsprocessingmodelerwidget.h

Reply:
But considering that the class name is `QgsProcessingModelerParameterWidget`, why the filename is not `qgsprocessingmodelerparameterwidget.h`?